client/layout/sidebar/index.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default React.createClass( {
+	displayName: 'Sidebar',
+
+	propTypes: {
+		className: React.PropTypes.string,
+		onClick: React.PropTypes.func
+	},
+
+	render: function() {
+		return (
+			<ul className="sidebar" onClick={ this.props.onClick }>
```

Comment:
I agree, but wanted to get the components in place first without changing the markup. I will loop back to it once I create sidebar-item.